ui/src/plugins/com.google.PerfettoMcp/styles.scss

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2025 The Android Open Source Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* Make the main container a flex column that fills the available height */
+.pf-ai-chat-panel {
+  display: flex;
+  flex-direction: column;
+  height: 95vh;
+  width: 100%;
+  top: 0;
+  background-color: #f7f7f7;
+  font-family: sans-serif;
+
+  /* This is the key part for scrolling:
+    - flex-grow: 1 allows this element to expand and fill available space.
+    - overflow-y: auto adds a scrollbar only when content overflows.
+  */
+  &__conversation {
+    flex-grow: 1;
+    overflow-y: scroll;
+    padding: 1rem;
+
+    /* Basic styling for messages to make them readable */
+    .pf-ai-chat-message {
+      margin-bottom: 1rem;
+      max-width: 80%;
+      padding: 0.5rem 1rem;
+      border-radius: 12px;
+      line-height: 1.5;
+
+      &__role-label {
+        font-weight: bold;
+        margin-right: 8px;
+        display: block;
+        margin-bottom: 4px;
+      }
+    }
+
+    .ai {
+      background-color: #e1e1e1;
+      align-self: flex-start;
+    }
+
+    .user {
+      background-color: #007bff;
+      color: white;
+      align-self: flex-end;
+      margin-left: auto;
+      /* Pushes the message to the right */
+    }
+
+    .error {
+      background-color: #f8d7da;
+      color: #721c24;
+      border: 1px solid #f5c6cb;
+    }
+
+    .thought {
+      background-color: #fff3cd;
+      color: #856404;
+      font-style: italic;
+      font-size: 0.9em;
+    }
+  }
+
+  /* Style for the footer input area */
+  &__input-area {
+    flex-shrink: 0;
+    /* Prevents the footer from shrinking */
+    height: 75px;
```

Comment:
I wouldn't bother setting the height - I'd just rely on the padding - basically add some vertical padding as well below. Fixed heights can cause issues if the size of the elements inside changes.

Also you might want to add a small gap - something like `gap: 4px;` to separate the button and the text box a little.